### ISSUE TITLE
Docs: Missing content for `createEntityAdapter`

### ISSUE THREAD

**Issue by pBouillon** [state-adapt/state-adapt#60]
The documentation seems to be missing for `createEntityAdapter` in the [Core Adapters page](https://state-adapt.github.io/adapters/core). The other sections of this page features a description of the adapters, how to use them and the source code:

![image](https://github.com/state-adapt/state-adapt/assets/22640284/5629ff7c-39a2-4afd-bef1-52f7395db4fd)

If I can be of any help in investigating these, please tell me!

**Comment by mfp22** [comment]
Oh yeah. For this one, the source code is long and not very pretty. I just put a link to the source code instead. Maybe I should make it look less broken. Maybe change it to `Please see [url] for source code.` 

What do you think?

**Comment by mfp22** [comment]
Oh and this is the bottom of those docs. They're there, just above where you're looking. It's just the source code section that looks broken.

**Comment by pBouillon** [comment]
Good idea! It might will also look consistent with other external references in other pages:

![image](https://github.com/state-adapt/state-adapt/assets/22640284/81f44db4-8af3-4260-aa5d-2b15d560e516)

**Comment by mfp22** [comment]
There are major changes to the adapter docs that will be published soon. They're in the codebase already. Just need to publish them. They will be more standard and clear.
